SparseGrids/tasgridExternalTests.cpp

```diff
@@ -89,13 +91,14 @@ TestResults ExternalTester::getError(const BaseFunction *f, TasGrid::TasmanianSp
     int num_outputs = f->getNumOutputs();
     int num_points = grid->getNumPoints();
     if ((type == type_integration) || (type == type_nodal_interpolation)){
-        double *points = grid->getPoints(), *weights = 0;
+        double *points = grid->getPoints();
+        double *weights = new double[num_points];
```

Comment:
Why assigning weights here? you are overwriting the pointer below.

Reply:
Reverted. I did this while trying to isolate some weird behavior, and I forgot to change it back.